Modules/Wishlist/Database/Migrations/2023_03_09_112150_create_wishlists_table.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wishlists', function (Blueprint $table) {
+            $table->id();
+            $table->tinyInteger('board_id')->default('0');
```

Comment:
Should be unsignedBigInteger and foreign key is required to board table

Reply:
Done